types/craco__craco/index.d.ts

```diff
@@ -0,0 +1,341 @@
+// Type definitions for @craco/craco 6.4
+// Project: https://github.com/gsoft-inc/craco
+// Definitions by: Chuck Fan <https://github.com/fanck0605>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 4.6
```

Comment:
```suggestion
```
so the tiself package was updated (released) 2 years ago, and you want to limit its usability in ecosystem to not yet released typescript version? Why is that?

Reply:
Can I use different $ExpectType for different typescript version?

``` typescript
// TypeScript 4.6
// $ExpectType { isFound: boolean; match: RuleSetRule | RuleSetUseItem; }
const matchedLoader = getLoader({}, matchesLessLoader);
```

``` typescript
// TypeScript 4.1
// $ExpectType { isFound: boolean; match: string | RuleSetRule | RuleSetLoader | ((data: any) => string | RuleSetLoader); }
const matchedLoader = getLoader({}, matchesLessLoader);
```

They are the same https://github.com/DefinitelyTyped/DefinitelyTyped/blob/3021fbb6c9a38d8eca28d793c1bc726652280097/types/webpack/v4/index.d.ts#L607...L610

Reply:
settled on 4.2